ipv-core/src/test/resources/localization/test.races.bad.csv

```diff
@@ -0,0 +1,18 @@
+American Indian or Alaska Native
+Asian
+Black or African American
+Hispanic or Latino
+Native Hawaiian or Other Pacific Islander
+White
```

Comment:
Should probably replace white with caucasian

Reply:
Actually, loking at the HL7 org, it doesn't even have caucasian so what's there is probably fine. 

https://www.hl7.org/fhir/v3/Ethnicity/cs.html

Reply:
"white" was the value used in the US federal register entry from which we got the race values.  These are excerpts from the shipped file.